.github/workflows/reusable-performance.yml

```diff
@@ -163,9 +167,21 @@ jobs:
           docker compose run --rm php php -i
           docker compose run --rm php locale -a
 
+      - name: Install object cache drop-in
+        if: ${{ inputs.memcached }}
+        run: cp src/wp-content/object-cache.php build/wp-content/object-cache.php
+
       - name: Install WordPress
         run: npm run env:install
 
+      - name: Enable themes on Multisite
+        if: ${{ inputs.multisite }}
+        run: |
+          npm run env:cli -- theme enable twentytwentyone --network --path=/var/www/${{ env.LOCAL_DIR }}
+          npm run env:cli -- theme enable twentytwentythree --network --path=/var/www/${{ env.LOCAL_DIR }}
+          npm run env:cli -- theme enable twentytwentyfour --network --path=/var/www/${{ env.LOCAL_DIR }}
+          npm run env:cli -- theme enable twentytwentyfive --network --path=/var/www/${{ env.LOCAL_DIR }}
```

Comment:
Not a blocker for this PR, but I'm wondering:
* Is it guaranteed that each of these themes is installed as part of WordPress Core installation?
* Can we somehow detect them / iterate through a list of all relevant themes and do this, instead of hard-coding the themes in this workflow (in addition to the actual specs)?

Reply:
All default themes are always in the repo, yes. That's where they are maintained.

Normally you could loop through all of them by combining WP-CLI commands, but because these go through npm scripts and Docker you have unwanted text like `Container wordpress-develop-mysql-1  Running` in the script output, so can't really use things like xargs.